eth/protocols/eth/peer.go

```diff
@@ -367,12 +344,6 @@ func (p *Peer) ReplyBlockHeaders(id uint64, headers []*types.Header) error {
 	})
 }
 
-// SendBlockBodiesRLP sends a batch of block contents to the remote peer from
```

Comment:
This was removed by upstream or by us? Just to check if this could be a pain in next merges

Reply:
By Upstream